src/ControllerV2/BeneficiaryResetPasswordController.php

```diff
@@ -17,7 +20,7 @@
 class BeneficiaryResetPasswordController extends AbstractController
 {
     #[IsGranted('UPDATE', 'beneficiary')]
```

Comment:
You can share this annotation, not sure if it is a good idea as it has a subject